integration_test/third_party_apps_data/applications/mssql/metadata.yaml

```diff
@@ -2,17 +2,22 @@ public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agen
 short_name: SQL Server
 long_name: Microsoft SQL Server
 description: |-
-  The Microsoft SQL Server integration collects metrics from your SQL Server
+  The v1 Microsoft SQL Server integration collects metrics from your SQL Server
   instances. The metrics provide transaction-rate information and also data on
   open connections.
 
+  The v2 Microsoft SQL Server integration collects metrics from your SQL Server
+  instances and databases. Page, batch, lock, and user connection count information
+  is provided at the instance level. Transaction and transaction log information
+  is provided at the database level. 
+
   For more information on SQL Server, see the
   [SQL Server](https://docs.microsoft.com/en-us/sql/sql-server)
   documentation.
 minimum_supported_agent_version:
-  metrics: 2.10.0
-  logging: 2.10.0
+  metrics: 2.15.0
```

Comment:
The minimum_supported_version will be auto filled by our release job as it’s hard to predict which version the new integration will be in. But it’s fine for this change as we are pretty sure it will be 2.15.0. 
another thing worth mentioning is the V1 mssql was supported since 1.x.x(thus 2.10.0 in the original file is wrong). It’s still not quite right to override the V1 minimum supported version with V2’s. That is another reason we might wanna another metadata_v2.yaml @igorpeshansky @jefferbrecht @qingling128